pkg/utils/confirmation.go

```diff
@@ -40,6 +40,13 @@ func AskForConfirm(reader io.Reader, writer printer.PrintService, message string
 		return errors.New("unable to parse users input " + err.Error())
 	}
 
+	//TODO: when responding no should not return error with invalid user input
+
+	//temporary solution
+	if answer == "n" || answer == "no" {
+		return errors.New("Stopped")
+	}
+
 	if answer != "y" && answer != "ye" && answer != "yes" {
 		return errors.New("invalid user input")
 	}
```

Comment:
This is a pretty funny code snippet. I think we should scrap this altogether. But, let's handle this in a different PR. Could you please revert this? You can leave a TODO flag here, if you wish